File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/queryvb/vb/createalinqquery5.vb
' <snippetcreatealinqquery5>


 Dim queryContacts = From c In svcContext.ContactSet _
                                    Where (c.Address1_City.Equals("Redmond")) _
                                    And (c.FirstName.Equals("Joe") _
                                            Or c.FirstName.Equals("John")) _
                                        Select New Contact With _
                                               {
                                                   .FirstName = c.FirstName,
                                                   .LastName = c.LastName,
                                                   .Address1_City = c.Address1_City
                                               }
 Console.WriteLine("Contacts in Redmond named Joe OR John")
 Console.WriteLine("=====================================")
 For Each c In queryContacts
  Console.WriteLine(c.FirstName &amp; " " &amp; c.LastName &amp; " " &amp; c.Address1_City)
 Next c

' </snippetcreatealinqquery5>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/query/cs/createalinqquery5.cs
// <snippetcreatealinqquery5>


 var queryContacts = from c in svcContext.ContactSet
                     where (c.Address1_City == "Redmond") &&
                           (c.FirstName.Equals("Joe") ||
                            c.FirstName.Equals("John"))
                     select new Contact
                     {
                      FirstName = c.FirstName,
                      LastName = c.LastName,
                      Address1_City = c.Address1_City
                     };
 System.Console.WriteLine("Contacts in Redmond named Joe OR John");
 System.Console.WriteLine("=====================================");
 foreach (var c in queryContacts)
 {
  System.Console.WriteLine(c.FirstName + " " +
      c.LastName + " " + c.Address1_City);
 }

// </snippetcreatealinqquery5>